ang/afform/afsearchSiteTokens.aff.html

```diff
@@ -0,0 +1,9 @@
+<div class="af-markup">
+  <div class="help">
+  <div>{{:: ts('Site Tokens allow you to define pre-set snippets of content which you can then insert via CiviCRM\'s standard token-handling features. Site Tokens marked as "reserved" are also inserted automatically in specific places within CiviCRM features, such as Message Templates.')}} <a class="crm-doc-link no-popup" href="https://docs.civicrm.org/user/en/latest/common-workflows/tokens-and-mail-merge/#site" target="_blank">{{:: ts('Learn more ...') }}</a></div>
```

Comment:
I need to check the right way to add a link - this has the language hard-coded - maybe @colemanw  or @mlutfy knows